werkzeug/debug/console.py

```diff
@@ -175,12 +175,6 @@ def runsource(self, source):
             output = ThreadedStream.fetch()
         return prompt + escape(source) + output
 
-    def runcode(self, code):
```

Comment:
I confess I don’t fully understand what this method was overriding, and the [original commit](https://github.com/alexpantyukhin/werkzeug/commit/4ba78986452c322c95fe600409814e4c3db85040) is over nine years old.

I notice that the standard library version of this method uses `exec`, not `eval` (see [code.py L91](https://hg.python.org/cpython/file/tip/Lib/code.py#l91)). Swapping this out for `exec(code, self.locals)` does fix my initial problem, but introduces the same problem of empty tracebacks as this patch.